test/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 aux_source_directory(${CMAKE_CURRENT_SOURCE_DIR} TEST_SOURCES)
+if (APPLE)
+	# don't want to test mknod - requires super-user privileges
+	add_definitions(-DNO_MKNOD)
```

Comment:
May be it makes sense to add project prefix to this macro, like `FBR_TEST_NO_MKNOD`? Just to be safe.